doc/source/installation/apt_repo.rst

```diff
@@ -58,7 +58,7 @@ Installing |Percona Server| from Percona ``apt`` repository
 
       $ sudo dpkg -i percona-release_latest.$(lsb_release -sc)_all.deb
 
-   Once you install this package the Percona repositories should be added. You can check the repository setup in the :file:`/etc/apt/sources.list.d/percona-release.list` file.
+   Once you install this package the Percona repositories should be added. You can check the repository setup in the :file:`/etc/apt/sources.list.d/percona-original-release.list` file.
```

Comment:
Once you install this package, the Percona repositories

Reply:
fixed